resources/views/hr/slots/edit.blade.php

```diff
@@ -0,0 +1,51 @@
+@extends('layouts.app')
+@section('content')
+<div class="container">
+@include('status', ['errors' => $errors->all()])
+<div class="card">
+    <div class="card-header">
+       Edit Slot
+    </div>
+
+    <div class="card-body">
+        <form action="{{ route("hr.slots.update",$slot->id) }}" method="POST">
```

Comment:
just another way of doing it
```suggestion
        <form action="{{ route("hr.slots.update", $slot) }}" method="POST">
```